app/components/moderation-list-row/component.js

```diff
@@ -49,15 +69,11 @@ export default Component.extend({
     }),
 
     // translations
-    dateLabel: computed('submission.reviewsState', 'gtDay', function() {
+    statusTimeDate: computed('submission.reviewsState', 'gtDay', 'noActions', function() {
+        const i18n = this.get('i18n');
         const dayValue = this.get('gtDay') ? 'gtDay' : 'ltDay';
-        return ACTION_LABELS[this.get('submission.reviewsState')][dayValue];
-    }),
-
-    submittedByLabel: computed('submission.reviewsState', function() {
-        return this.get('submission.reviewsState') === PENDING ?
-            'components.moderation-list-row.submission.by' :
-            'components.moderation-list-row.submission.submission_by';
+        const timeWording = this.get('noAction') ? `${dayValue}_automatic` : dayValue;
+        return i18n.t(ACTION_LABELS[this.get('submission.reviewsState')][timeWording], { timeDate: this.get('relevantDate'), moderatorName: this.get('lastActionCreator') });
```

Comment:
Could you split this line up somehow? Maybe move `ACTION_LABELS[...` into a variable?